docs/users-guide/writing-sql.md

```diff
@@ -0,0 +1,39 @@
+## Asking more advanced questions in the SQL/native query editor
+
+---
+
+If you ever need to ask questions that can't be expressed using the question builder, you can use **SQL** instead.
+
+### What's SQL?
+
+SQL (pronounced "sequel") stands for Structured Query Language, and is a widely used standard for getting data from databases. We won't try to teach you all about SQL in this guide, but to learn more about it, check out this [SQL Tutorial](http://www.w3schools.com/sql/default.asp).
+
+Even if you don't understand SQL or how to use it, it's worthwhile to understand how works inside Metabase because sometimes other people will share SQL-based questions that might be useful to you.
+
+### Starting a new SQL query
+
+If you have the [permissions](../administration-guide/05-setting-permissions.md) to use the SQL editor, when you click the Ask a Question button you'll see an option to start a new SQL query. There's also a shortcut to the editor in the top nav bar; it's the little console icon.
+
+You can write SQL (or your database's native querying language) directly into the editor that appears.
+
+![SQL editor](images/SQLInterface.png)
+
+To try it out, type the command `select sum(subtotal), created_at from orders group by created_at`. Don't worry if you don't understand this just yet. Click the blue run button and note the table that comes back is the same as if you had asked for the sum of Subtotal in the Orders table, grouped by Created At.
```

Comment:
Should we specify it's a blue Get Answer button instead of a Run button?

Reply:
This actually isn't the case — in `release-0.33.x` we've changed it to be a run/play button (triangle icon) instead of "Get Answer" when writing SQL.